doc/security.md

```diff
@@ -37,7 +37,7 @@ Introduce and enable this library in your SpringBoot application by adding the `
 
 ```java
 @SpringBootApplication
-@EnableCustomSecurity
+@EnableCustomSecurity(applicationBasePathPattern = "/api/**")
```

Comment:
I think it is better to define a default value for this parameter and make it optional. The default value can be all paths: "/**"